Marlin/src/module/planner.h

```diff
@@ -1023,6 +1043,14 @@ class Planner {
      */
     static block_t* get_current_block();
 
+    /**
+     * Get an planned upcomming block from the buffer.
+     * Return nullptr if the buffer doesn't have the `current + offset` yet .
+     *
+     * WARNING: Called from Stepper ISR context!
+     */
+    static block_t* lookahead(uint8_t offset);
```

Comment:
I think `lookahead()` is a pretty generic name. Maybe `get_future_block()` instead?

Reply:
renamed to get_future_block